docs/api-reference/geo-layers/mvt-layer.md

```diff
@@ -161,6 +165,26 @@ new MVTLayer({
 })
 ```
 
+* `getVisibleTiles` (async Function)
```

Comment:
`getVisibleTiles` (Function)

Arguments: ...

Returns: a promise that resolves to ...